scripts/tests/chiptest/test_definition.py

```diff
@@ -239,16 +240,20 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
             # so it will be commissionable again.
             app.factoryReset()
             app.start(str(randrange(1, 4096)))
-
-            runner.RunSubprocess(
-                tool_cmd + ['pairing', 'qrcode', TEST_NODE_ID, app.setupCode] +
-                ['--paa-trust-store-path', DEVELOPMENT_PAA_LIST],
-                name='PAIR', dependencies=[apps_register])
-
+            pairing_cmd = tool_cmd + ['pairing', 'qrcode', TEST_NODE_ID, app.setupCode]
+            if sys.platform != 'darwin':
```

Comment:
Won't this affect regular chip-tool on darwin too?

Followup OK by me to sort this out. Maybe chip-tool-darwin should take the `--paa-trust-store-path` arg  but just ignore it....

Reply:
Yes, this will affect chip-tool on Darwin. I will do a followup to remove this argument completely since chip-tool defaults to the [same path](https://github.com/project-chip/connectedhomeip/blob/master/examples/chip-tool/commands/common/CHIPCommand.cpp#L96) provided.

Reply:
Ok. Created an issue [here](https://github.com/project-chip/connectedhomeip/issues/17366).